src/views/TeamCreationView.vue

```diff
@@ -1,7 +1,6 @@
 <template>
   <div class="container">
-    <h1>Create a team</h1>
-
+    <page-title :title="'Create a Team'" />
```

Comment:
Are the double quotes ("') here intentional?

Reply:
What do you mean? The double quotes on :title=""? I don't know of any other way to pass data with props in this situation, and if you're passing a string directly you just wrap the inner ones in single quotes. Trying something like :title='Create a Team' will throw an error

Reply:
You have a `"` then a `'` right now. Should it just be a `"` or did you mean to also include the `'`?

Reply:
Yeah you need the '' to pass a String unless I missed some shortcut. Just doing :title="Create a Team" was giving me an error